packages/flutter/test/widgets/interactive_viewer_test.dart

```diff
@@ -623,6 +623,43 @@ void main() {
       expect(transformationController.value.getMaxScaleOnAxis(), equals(1.0));
     });
 
+
+    testWidgets('Scale with mouse returns onInteraction properties',(WidgetTester tester) async{
+      final TransformationController transformationController = TransformationController();
+      double currentScale;
+      Velocity currentVelocity;
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: Center(
+              child: InteractiveViewer(
+                transformationController: transformationController,
+                onInteractionStart: (ScaleStartDetails details){
```

Comment:
Maybe add a `calledStart` boolean and set it to true in here, then expect it to be true later?  Just to get onInteractionStart some test coverage, even though it doesn't do much.